packages/template-retail-react-app/app/components/_app/index.jsx

```diff
@@ -98,7 +124,7 @@ Learn more with our localization guide. https://sfdc.co/localization-guide
     const currency = locale.preferredCurrency || l10n.defaultCurrency
 
     // Set up customer and basket
-    useShopper({currency})
+    // useShopper({currency})
```

Comment:
❔

Reply:
This hook is for the old implementation, it is safe to delete